static/css/font-source-sans-pro.css

```diff
@@ -2,27 +2,27 @@
     font-family:"Source Sans Pro";
     font-style:normal;
     font-weight:300;
-    font-display:auto;
+    font-display:swap;
```

Comment:
Why swap? I assume responsiveness, but is it a best practice or solving a specific problem?